src/Chart.py

```diff
@@ -1,7 +1,7 @@
 from sensors.sensor import Sensor
 
 class Chart():
-    def __init__(self, sensor: Sensor, AlarmList):
```

Comment:
i already fixed this in a recent PR, not sure if it will cause conflicts, but im guessing not

Reply:
I rebased it so it should be fine